decentraland/unity-explorer
# Fix: fixed backpack grid audio issues

## What does this PR change?

Fix #800 
Removes unneded audio clips and fixes auto playback on panel switching

...

## How to test the changes?



1. Launch the explorer
2. Go to backpack in explore panel
3. Switch to another tab
4. Verify that there are no strange audios related to backpack slots played

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
